src/Mapbender/WmsBundle/Component/WmsMetadata.php

```diff
@@ -26,37 +26,36 @@ private function prepareData($itemName)
         if ($this->getUseCommon()) {
             $source_items = array();
             $source_items[] = array("title" => SourceMetadata::getNotNull($src->getTitle(), $this->instance->getTitle()));
-            $source_items[] = array("name" => SourceMetadata::getNotNull($src->getName()));
-            $source_items[] = array("version" => SourceMetadata::getNotNull($src->getVersion()));
-            $source_items[] = array("originUrl" => SourceMetadata::getNotNull($src->getOriginUrl()));
-            $source_items[] = array("description" => SourceMetadata::getNotNull($src->getDescription()));
-            $source_items[] = array("onlineResource" =>
-                SourceMetadata::getNotNull($src->getOnlineResource() !== null ? $src->getOnlineResource() : ""));
-            $source_items[] = array("exceptionFormats" => SourceMetadata::getNotNull(implode(",", $src->getExceptionFormats())));
+            $source_items[] = array("name" => strval($src->getName()));
```

Comment:
strval(null) == ''
And this is all `getNotNull` ever achieved when called with a single argument.